test/spec/features/copy-paste/BpmnCopyPasteSpec.js

```diff
@@ -378,8 +378,8 @@ describe('features/copy-paste', function() {
 
           var taskBo = getBusinessObject(task);
 
-          expect(taskBo.di.fill).to.equal(fill);
-          expect(taskBo.di.stroke).to.equal(stroke);
+          expect(taskBo.di.get('background-color')).to.equal(fill);
```

Comment:
Please continue to verify existing `bioc` utility.

Reply:
I see. Moved to legacy test cases: https://github.com/bpmn-io/bpmn-js/pull/1453/files#diff-9e06a6ec04edd012b42f08a04172bd1814bdd97bdd277158bfe61e467541bb9aR560

Reply:
Done.